src/index.js

```diff
@@ -708,24 +716,44 @@ class LocalstackPlugin {
    *                    a rejected promise on any connection error.
```

Comment:
The documentation does not match the updated busy-wait approach. Maybe adding a "synchronous" busy-wait wrapper would be clearer.